pyccel/ast/numpyext.py

```diff
@@ -21,35 +20,43 @@
 from .core import PyccelEq,  PyccelNe,  PyccelLt,  PyccelLe,  PyccelGt,  PyccelGe
 from .core import PyccelAnd, PyccelOr,  PyccelNot, PyccelMinus, PyccelAssociativeParenthesis
 
-from .builtins       import Int as PythonInt
-from .builtins       import PythonFloat, PythonTuple, Complex as PythonComplex
+from .builtins       import Int as PythonInt, Bool as PythonBool
+from .builtins       import PythonFloat, PythonTuple, PythonComplex
 from .datatypes      import dtype_and_precision_registry as dtype_registry
 from .datatypes      import default_precision
 from .datatypes      import datatype
 from .datatypes      import NativeInteger, NativeReal, NativeComplex, NativeBool
 from .numbers        import Integer, Float
-from .type_inference import sp_dtype, str_dtype
+from .type_inference import str_dtype
 
 
 __all__ = (
-    'Abs',
-    'Acos',
-    'Acot',
-    'Acsc',
-    'Array',
-    'Asec',
-    'Asin',
-    'Atan',
-    'Bounds',
-    'Complex',
-    'Complex64',
-    'Complex128',
```

Comment:
Did we remove some names by accident? `Complex64` and `Complex128` should probably be added back to the list (next to `Float32` and `Float64`)

Reply:
yes we should add them